entrypoint.py

```diff
@@ -282,13 +330,18 @@ def main():
     else:
         SBOM_FILE = None
 
+    if os.getenv("DOCKER_IMAGE"):
```

Comment:
Isn't this block same as just doing

```python
DOCKER_IMAGE = os.getenv("DOCKER_IMAGE")
```

base `os.getenv` would return None if the env var is not found?